code/game/gamemodes/wizard/wizard.dm

```diff
@@ -64,9 +64,17 @@
 
 /datum/game_mode/wizard/set_round_result()
 	..()
-	if(finished)
+	var/didEscape = TRUE
+	for(var/datum/mind/wizard in wizards)
+		if(considered_escaped(wizard))
+			didEscape = FALSE
```

Comment:
You can cast your proc like so
```suggestion
	var/didEscape = TRUE
	var/datum/objective/O
	for(var/datum/mind/wizard in wizards)
		if(O.considered_escaped(wizard))
			didEscape = FALSE
```
This allows you to access /datum/objective/ procs even if you aren't in that file.